public/keymaps/p/pom_keyboards_tnln95_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "pom_keyboards/tnln95",
+  "keymap": "default",
+  "commit": "cf141c00d636b67179f7b9436284fbd0b7f6def2",
+  "layout": "LAYOUT_ansi",

```

Update commit hash and change the layout macro.

```suggestion
  "commit": "1b4f47c8ee099f373dddd5b294e09bd78b39055f",
  "layout": "LAYOUT_all",
```